### ISSUE TITLE
Getting started revamp

### ISSUE THREAD

**Issue by DubhAd** [home-assistant/home-assistant.github.io#6555]
A starting point for story boarding what the Getting Started should look like.

Starting thought:

1. Explain what HA is
2. Install guide (Hass.io focussed, or just link to the Hass.io guide?)
3. Configuration
   - create first user (owner account)
   - timezone, location, etc
   - add-ons (Hass.io)
   - onboarding users
4. First automation - using the UI, but with a link to the automation docs
5. Remote access using DuckDNS and LetsEncrypt
   - Include call out to the securing guide
6. Presence detection using OwnTracks HTTP
7. Automation using presence detection?
8. Things to know (validation, reloading, etc)
9. Where to get help
   - Pointer to the component pages
   - Pointer to the docs
   - Pointer to the YAML part of the docs
   - Forum
   - Discord

**Comment by nwesterhausen** [comment]
Right now on Getting Started, it goes right into installing with Hass.io. I like the way the [installation documentation](https://www.home-assistant.io/docs/installation/) says "here are our three recommended install options." I think the Getting Started Install Guide should give three options, linking to their respective guide pages. However, I think it should have Hass.io, Docker, and Python 3.5 instructions linked (instead of linking to both Hass.io and Hassbian).

We can still recommend Hass.io for total beginners there, but Docker and Python installs are both valid options that someone coming to check out Home Assistant may be more comfortable following. For example:
- Personally, since I have a server running Docker, if I'm looking to try something out, it's a lot easier (and quicker) to run a Docker command than to image an SD card for a pi and get that running. 
- Someone who has a Raspberry pi already running some services likely doesn't want to wipe that pi out. The Python instructions allow Home Assistant to be run on a server that is already set up.

In short, if someone has a pi and SD card ready, they should follow Hass.io instructions. If they don't have a pi around but do have an existing server, then the Docker or Python instructions can be followed. I think that Getting Started should reflect this.

**Comment by balloob** [comment]
I disagree @nwesterhausen because as a project we're thinking bigger. I don't want users with barely any admin experience to consider any other option but hass.io. People like to present all options as equal but they are not. We have developed hass.io so people don't have to worry about security, updating and can focus on automating their house. So the first step should be Hass.io but mention that there are other options for advanced users. 

Tinkerer you have my full blessing to make this happen. If it helps, add different issues to make different pages. Some pages are evolution of what we have today, some are brand new. We can update the getting started as pages are finished.
